include/base.php

```diff
@@ -163,7 +162,7 @@ public function switch_blog( $new_blog_id, $prev_blog_id ) {
 	 * @return bool
 	 */
 	protected function is_active_on_current_site(): bool {
-		return pll_is_plugin_active( POLYLANG_BASENAME ) && get_option( 'polylang' );
+		return pll_is_plugin_active( POLYLANG_BASENAME );
```

Comment:
Not sure we should remove `get_option()` here, if I remember correctly it's here to really ensure there are options in DB for the current site for extra safety.

Reply:
Fixed in [37ee8c9](https://github.com/polylang/polylang/pull/1481/commits/37ee8c9e8a61d2efd62e24dd25c76c47e1f3333c).